sdk/sdk-core/aar/src/main/java/mil/emp3/core/storage/StorageManager.java

```diff
@@ -897,7 +897,7 @@ public void addOverlays(IMap map, List<IOverlay> overlays, boolean visible) thro
             this.executeTransaction(transactionList);
         } finally {
             lock.unlock();
-            eventManager.generateContainerEvent(ContainerEventEnum.OBJECT_ADDED, map, overlays);
+            eventManager.generateContainerEvent(ContainerEventEnum.OBJECT_ADDED, map, overlays, object);
```

Comment:
This is an existing bug. We are generating an event if actual action failed. This needs to be fixed in all place.